internal/lookout/configuration/types.go

```diff
@@ -60,10 +73,6 @@ type LookoutIngesterConfiguration struct {
 	BatchSize int
 	// Maximum time since the last batch before a batch will be inserted into the database
 	BatchDuration time.Duration
-	// Time for which the pulsar consumer will wait for a new message before retrying
```

Comment:
These two config options were unused by lookoutingester code. The equivalent options in LookoutIngesterConfiguration.Pulsar were used instead.

Reply:
good spot- yes I moved them when I went to the common framework code but forgot to remove the config options